test/polytopes.jl

```diff
@@ -448,7 +443,7 @@
     points = P2[(1,1),(2,2),(2,2),(3,3),(1,1)]
     poly   = PolyArea(points)
     unique!(poly)
-    @test first(chains(poly)) == Chain(points)
+    @test first(chains(poly)) == Chain(P2[(1,1),(2,2),(3,3),(1,1)])
```

Comment:
this was a buggy test?

Reply:
An auto-closing constructor has to either modify the input array with pop! (doesn’t work for inputs that are static or views), store a `@view points[begin:end-1]` internally, or create a copy of `points[begin:end-1]`. Storing the view makes a function like `unique!` impossible, so I went with the last option. The test relies on the fact that both the original `points` array and the internal array of the Chain are modified by `unique!`, which is no longer the case.